zaas-service/src/main/java/org/zowe/apiml/zaas/health/ZaasHealthIndicator.java

```diff
@@ -32,29 +29,16 @@
 public class ZaasHealthIndicator extends AbstractHealthIndicator {
     private final DiscoveryClient discoveryClient;
     private final Providers loginProviders;
-    private String apiCatalogServiceId;
-    
-    private final ApimlLogger apimlLog = ApimlLogger.of(ZaasHealthIndicator.class,
-            YamlMessageServiceInstance.getInstance());
-    boolean startedInformationPublished = false;
 
     public ZaasHealthIndicator(DiscoveryClient discoveryClient,
-                               Providers providers,
-                               @Value("${apiml.catalog.serviceId:}") String apiCatalogServiceId) {
+                               Providers providers) {
         this.discoveryClient = discoveryClient;
         this.loginProviders = providers;
-        this.apiCatalogServiceId = apiCatalogServiceId;
     }
 
 
     @Override
     protected void doHealthCheck(Health.Builder builder) {
-        boolean anyCatalogIsAvailable = apiCatalogServiceId != null && !apiCatalogServiceId.isEmpty();
-        boolean apiCatalogUp = !this.discoveryClient.getInstances(apiCatalogServiceId).isEmpty();
-
-        // When DS goes 'down' after it was already 'up', the new status is not shown. This is probably feature of
-        // Eureka client which caches the status of services. When DS is down the cache is not refreshed.
-        boolean discoveryUp = !this.discoveryClient.getInstances(CoreService.DISCOVERY.getServiceId()).isEmpty();
 
         boolean authUp = true;
         if (loginProviders.isZosfmUsed()) {
```

Comment:
I think the `System.exit(-1)` a couple of lines below should be removed and replaced by unhealthy response